src/libraries/System.Private.CoreLib/src/System/IO/File.cs

```diff
@@ -181,86 +181,445 @@ internal static DateTimeOffset GetUtcDateTimeOffset(DateTime dateTime)
         public static void SetCreationTime(string path, DateTime creationTime)
             => FileSystem.SetCreationTime(Path.GetFullPath(path), creationTime, asDirectory: false);
 
+        /// <summary>
+        ///     Sets the date and time the file was created.
```

Comment:
```suggestion
        /// Sets the date and time the file was created.
```
triple-slash docs shall not be indented.